Libraries/DirectMLX.h

```diff
@@ -3193,6 +3193,9 @@ namespace dml
         Optional<Expression> bias,
         Span<const uint32_t> axes,
         bool normalizeVariance,
+#if DML_TARGET_VERSION >= 0x6300
+        bool normalizeMean,
+#endif
         float epsilon,
         FusedActivation fusedActivation = FusedActivation::None())
     {
```

Comment:
Why not just have 2 functions 

have one called MeanVarianceNormalization2

Reply:
That's just the precedent for DMLX, that we don't have named variants.